data/overview.yaml

```diff
@@ -115,69 +115,69 @@ Groups:
-  Bezouts theorem:
+  Bezout's theorem: ''
   Euclidean rings: 'algebra/euclidean_domain.html#euclidean_domain'
-  Euclid's' algorithm:
-  Z is an euclidean ring:
-  K[X] is an euclidean ring when K is a field:
-  Irreducible polynomial:
-  Cyclotomic polynomials in Q[X]:
-  Eisenstein's criterion: 'number_theory/quadratic_reciprocity.html#zmod.eisenstein_lemma' # FIXME: cheat?
-  Congruence in Z:
+  Euclid's' algorithm: 'data/int/gcd.html#nat.xgcd'
+  Z is a euclidean ring: 'algebra/euclidean_domain.html#int.euclidean_domain'
+  K[X] is a euclidean ring when K is a field: 'data/polynomial.html#polynomial.euclidean_domain'
+  Irreducible polynomial: 'algebra/associated.html#irreducible'
+  Cyclotomic polynomials in Q[X]: ''
+  Eisenstein's criterion: ''
+  Congruence in Z: 'data/int/modeq.html#int.modeq'
   Prime numbers: 'algebra/associated.html#associates.prime' 
-  Z/nZ and its invertible elements:
-  Euler's indicator function (phi):
+  Z/nZ and its invertible elements: 'data/zmod/basic.html#zmod.unit_of_coprime'
+  Euler's indicator function (phi): 'data/nat/totient.html#nat.totient'
 Polynomial:
-  Polynomial algebra on one or several indeterminates over a commutative ring:
+  Polynomial algebra on one or several indeterminates over a commutative ring: 'data/mv_polynomial.html#mv_polynomial'
   Roots of a polynomial: 'data/polynomial.html#polynomial.roots'
   Multiplicity: 'data/polynomial.html#polynomial.root_multiplicity'
   Relationship between the coefficients and the roots of a split polynomial:
   Newton's sums:
-  Derived polynomials:
+  Polynomial derivative: 'data/polynomial.html#polynomial.derivative'
   Decomposition into sums of homogeneous polynomials:
   Symmetric polynomials:
 Fields:
   fields:
     Characteristics:
       Characteristic of a ring: 'algebra/char_p.html#ring_char'
-      Characteristic zero: 'algebra/char_p.html#char_p'
-      Characteristic p: 'algebra/char_zero.html#char_zero'
+      Characteristic zero: 'algebra/char_zero.html#char_zero'
+      Characteristic p: 'algebra/char_p.html#char_p'
   subfields: 'field_theory/subfield.html'
   Frobenius morphisms: 'algebra/char_p.html#frobenius' 
   field extensions:
   integral ring fractional fields:
   Field Q of rational numbers: 'data/rat/basic.html#rat.division_ring'
   Field R of real numbers: 'data/real/basic.html#real.division_ring'
   Field C of complex numbers: 'data/complex/basic.html#complex.field
-  D'Alembert-Gauss theorem: 'analysis/complex/polynomial.html#complex.exists_root'
+  "D'Alembert-Gauss theorem": 'analysis/complex/polynomial.html#complex.exists_root'
   Algebraic elements:
   Transcendental elements:
   Algebraic extensions:
   Algebraically closed fields:
-  Rupture fields:
+  Rupture fields: 'ring_theory/adjoin_root.html#adjoin_root'
   Splitting fields:
   Finite fields: 'field_theory/finite.html'
   Rational fraction fields with one indeterminate over the field:
```

Comment:
Isn't this offered by fraction fields? I guess it's fair to wait until we have some convenient API anyway.